README.md

```diff
@@ -9,7 +9,7 @@ following diagram provides an overview of the collection process.
 
 ## Installation
 
-Detailed instructions are available in our Installation Guides in documentation below.
+Detailed instructions are available in our [Installation Guides](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/docs/installation.md).
```

Comment:
```suggestion
Detailed instructions are available in our [Installation Guides](/docs/installation.md).
```
Absolute repo path is better in my opinion, then it won't matter which branch the reader is on.